js/src/dom/eventHandler.js

```diff
@@ -23,6 +23,19 @@ const workingDefaultPrevented = (() => {
   return e.defaultPrevented
 })()
 
+const defaultPreventedPreservedOnDispatch = typeof window.CustomEvent !== 'function' || (() => {
```

Comment:
to avoid double check of `typeof window.CustomEvent` you should move this IIFE where we add a polyfill to add `CustomEvent`

Reply:
and can you add a comment to explain the reason we added that please

Reply:
Ok, done